CHANGELOG.md

```diff
@@ -1,4 +1,22 @@
-## 2.19.0 (Unreleased)
+## 2.19.1 (Unreleased)
+### Notes:
+- Supported Controller version: **UserConnect-6.4.2499**
+- Supported Terraform version: **v0.12.x**, **v0.13.x** and **v0.14.x**
+
+### Features:
+#### Accounts
+1. Implemented support for separate IAM role and policy for gateways in AWSGov **aviatrix_account** and data source with new attributes
```

Comment:
New attributes are for IAM role based accounts in AWSGov because our provider didn't allow IAM based accounts in AWSGov. It's not for the separate IAM gateway roles.